src/app/modules/dashboard/components/validator-performance-summary/validator-performance-summary.component.ts

```diff
@@ -0,0 +1,115 @@
+  connectedPeers: number;
+  totalPeers: number;
+}
+
+@Component({
+  selector: 'app-validator-performance-summary',
+  templateUrl: './validator-performance-summary.component.html',
+  styles: [
+  ]
+})
+export class ValidatorPerformanceSummaryComponent {
+  constructor(
+    private validatorService: ValidatorService,
+    private walletService: WalletService,
+    private beaconNodeService: BeaconNodeService,
+  ) { }
+
+  tooltips = {
+    effectiveBalance: 'Describes your average validator balance across your active validating keys',
+    inclusionDistance: `This is the average number of slots it takes for your validator's attestations to get included in blocks. The lower this number, the better your rewards will be. 1 is the optimal inclusion distance`,
+    recentEpochGains: `This summarizes your total gains in ETH over the last epoch (approximately 6 minutes ago), which will give you an approximation of most recent performance`,
+    correctlyVoted: `The number of times in an epoch your validators voted correctly on the chain head vs. the total number of times they voed`,
+    keys: `Total number of active validating keys in your wallet`,
+    score: `A subjective scale from Perfect, Great, Good, to Poor which qualifies how well your validators are performing on average in terms of correct votes in recent epochs`,
+    connectedPeers: `Number of connected peers in your beacon node`,
+    totalPeers: `Total number of peers in your beacon node, which includes disconnected, connecting, idle, peers`,
+  };
+
+  validatingKeys$ = this.walletService.validatingPublicKeys$;
+  peers$ = this.beaconNodeService.peers$.pipe(
+    map(p => p.peers),
+    shareReplay(1),
+  );
+  connectedPeers$ = this.peers$.pipe(
+    map(peers => peers.filter(p => p.connectionState.toString() === 'CONNECTED')), 
+  );
+  performanceData$: Observable<PerformanceData> = this.validatorService.performance$.pipe(
+    map(this.transformPerformanceData.bind(this)),
+  );
+
+
+  private transformPerformanceData(perf: ValidatorPerformanceResponse): PerformanceData {
+    const recentEpochGains = this.computeEpochGains(
+      perf.balancesBeforeEpochTransition, perf.balancesAfterEpochTransition,
+    );
+    const averageEffectiveBalance = this.computeAverageEffectiveBalance(
+      perf.currentEffectiveBalances
+    );
+    const votedHeadPercentage = perf.correctlyVotedHead.filter(Boolean).length / 
+      perf.correctlyVotedHead.length;
+
+    const averageInclusionDistance = perf.inclusionDistances.reduce((prev, curr) => {
+      if (curr.toString() === FAR_FUTURE_EPOCH) {
+        return prev;
+      }
+      return prev + Number.parseInt(curr.toString());
+    }, 0) / perf.inclusionDistances.length;
+    let overallScore;
+    if (votedHeadPercentage === 1) {
+      overallScore = 'Perfect';
+    } else if (votedHeadPercentage >= 0.95) {
+      overallScore = 'Great';
+    } else if (votedHeadPercentage >= 0.80 && votedHeadPercentage < 0.95) {
```

Comment:
The `&& votedHeadPercentage < 0.95` part is not needed, you've already got this covered in the previous case.